CoffeeChat/Controller/OnboardingPageViewController.swift

```diff
@@ -46,7 +48,8 @@ class OnboardingPageViewController: UIPageViewController {
         demographicsViewController = DemographicsViewController(delegate: self)
         interestsViewController = InterestsViewController(delegate: self)
         groupsViewController = GroupsViewController(delegate: self)
-        onboardingPages = [demographicsViewController, interestsViewController, groupsViewController]
+        goalsViewController = GoalsViewController(delegate: self)
+        onboardingPages = [demographicsViewController, interestsViewController, groupsViewController, goalsViewController]
```

Comment:
Maybe split this on multiple lines since it is getting kinda long:
```
        onboardingPages = [
            demographicsViewController,
            interestsViewController,
            groupsViewController,
            goalsViewController
        ]
```